docs/docs/quickstart/real-world-example/README.md

```diff
@@ -0,0 +1,246 @@
+# Real-world Twitter -> Slack
+
+For the last example in this quickstart, we'll use many of the patterns introduced in [earlier examples](/quickstart) to solve a real-world use case and will cover how to:
+
+- [Trigger a workflow anytime @pipedream is mentioned on Twitter](#trigger-a-workflow-anytime-pipedream-https-twitter-com-pipedream-is-mentioned-on-twitter)
+- [Construct a message in Node.js using Slack Block Kit](#construct-a-message-in-node-js-using-slack-block-kit)
+- [Use an action to post the formatted message to a Slack channel](#use-an-action-to-post-the-formatted-message-to-a-slack-channel)
+- [Run a live test](#run-a-live-test)
+
+Following is an example of a Tweet that we'll richly format and post to Slack:
+
+![image-20210518194229746](./images/image-20210518194229746.png)
+
+::: tip
+If you didn't complete the previous examples, we recommend you start from the [beginning of this guide](/quickstart/) to learn the basics of workflow development. If you have any issues completing this example, you can [view, copy and run a completed version](https://pipedream.com/@gettingstarted/quickstart-real-world-twitter-slack-p_n1Co2dO).
+:::
+
+### Trigger a workflow anytime [`@pipedream`](https://twitter.com/pipedream) is mentioned on Twitter
+
+First, create a new workflow and select the **Twitter** app:
+
+![image-20210518190544989](./images/image-20210518190544989.png)
+
+Select **Search Mentions** to trigger your workflow every time a new Tweet matches your search criteria:
```

Comment:
Small nit but I don't believe "Tweet" is typically capitalized

Reply:
When referring to Tweets on Twitter, I found it is typically capitalized (including by Twitter)